File: ViewBase/Source/Validators/NullObjectValidator.vb
Imports FluentValidation

Public Class NullObjectValidator
    Inherits AbstractValidator(Of Object)
    Sub New(propertyDisplayName As String)
        RuleFor(Function(o) o).
            Must(Function(o) o IsNot Nothing).WithMessage($"{propertyDisplayName}: no puede ser nulo.")
    End Sub
End Class

File: CoreView/src/Validators/NullObjectValidator.cs
using FluentValidation;

namespace Wells.CoreView.Validators
{
    public class NullObjectValidator : AbstractValidator<object>
    {
        public NullObjectValidator(string propertyDisplayName)
        {
            RuleFor((o) => o).Must((o) => o != null).WithMessage($"{propertyDisplayName}: no puede ser nulo.");
        }
    }
}
